doc/onbaording/usage.md

```diff
@@ -0,0 +1,200 @@
+---
+title: 'Usage'
+weight: 40
+---
+
+## Getting Started with the Interface
+
+When you first open AsyncAPI Studio, you'll see the main editor interface with the following components:
+
+1. **Editor Panel**: The central area where you write and edit your AsyncAPI document  
+   ![Screenshot: Editor Panel](images/editor-panel.png)
+2. **Navigation Sidebar**: Access different views and tools  
+   ![Screenshot: Navigation Sidebar](images/navigation-panel.png)
+3. **Preview Panel**: View documentation, visual diagrams, or generated code  
+   ![Screenshot: Preview Panel](images/preview-panel.png)
+4. **Toolbar**: Access common actions like validation, saving, and sharing 
+
+## Creating a New AsyncAPI Document
+
+To create a new AsyncAPI document:
+
+1. Click on the **New Document** button in the toolbar  
+   ![Screenshot: New Document button](images/creating-new-document-1.png)
+2. Select the AsyncAPI version you want to use (2.0.0, 2.1.0, 2.2.0, etc.)  
+   ![Screenshot: Version selection](images/creating-new-document-2.png)
+3. Choose a template to start with (Minimal, Kafka, MQTT, etc.) or start from scratch  
+   ![Screenshot: Template selection](images/creating-new-document-3.png)
+4. Begin editing your document in the editor panel  
+   ![Screenshot: Editor panel with new document](images/creating-new-document-4.png)
+
+## Editing AsyncAPI Documents
+
+The Studio editor provides several features to help you create and edit AsyncAPI documents:
+
+### Syntax Highlighting
+
+The editor automatically highlights different parts of your AsyncAPI document to make it easier to read and understand:
+
+- Keywords and property names
+- Values and types
+- References and components
+- Comments and annotations
+
+![Screenshot: Syntax Highlighting](images/editing-syntax-highlighting.png)
+
+### Code Completion
+
+As you type, the editor suggests completions based on the AsyncAPI specification:
+
+1. Press **Ctrl+Space** (or **Cmd+Space** on Mac) to manually trigger suggestions  
+   ![Screenshot: Code Completion](images/editing-code-completion.png)
+2. The editor will suggest property names, values, and schema components based on the context
+
+### Error Detection
```

Comment:
There is a diagnostics panel in bottom as well. It has filter, navigation to error and other features as well.